beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -263,7 +264,7 @@ func ProcessValidatorRegistry(
 	}
 	mix, err := helpers.RandaoMix(state, randaoMixSlot)
 	if err != nil {
-		return nil, fmt.Errorf("could not get randao mix: %v", err)
+		return nil, fmt.Errorf("could not get seed mix: %v", err)
```

Comment:
Why is this renamed if the method is still randaoMix?

Reply:
More I think, randao is better name than seed, randao implies how the randomness gets generated where seed is just implies the source for randomness.

Changing back got randao